### PR TITLE
Adding Login Functionality

### LINKED ISSUE #83: Add Login Functionality
Login Functionality can be added in iOS app a it is added in android app.

Would you like to work on this issue?
Yes

### PR DESCRIPTION
**Pull request by SIMRAN1**
Fixes #83 
Login Functionality is added.Now user can Login through app.

### CODE REVIEW

**Comment by SIMRAN1** [comment]
@chashmeetsingh @imjog please review!

File: FOSSAsia/Client.swift [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            case .success(_):
+                if let data = response.result.value as? Dictionary<String, Any> {
+                    
+                    completion(data as AnyObject?, nil)
+                } else {
+                    completion(nil, NSError(domain: Constants.ResponseMessages.ServerError, code: 1))
+                }
+                break
+                
+            case .failure(let error):
```
Comment (jogendra) [inline FOSSAsia/Client.swift]: replace `let error` with `_`

**Review by jogendra — CHANGES_REQUESTED** [review]
If I enter a wrong email address and password, it is not showing any success or failure message, just disabling the fields. Please look into it.

File: FOSSAsia/Client.swift (hunk 1 above)
Comment (SIMRAN1) [reply FOSSAsia/Client.swift]: okk @imjog !!

**Comment by SIMRAN1** [comment]
okkk @imjog I will resolve it!!

**Comment by SIMRAN1** [comment]
Done @imjog !

**Review by jogendra — APPROVED** [review]
Looks good to me! Good work :)

**Pull request merged**